modules/assim.batch/R/minimize.GP.R

```diff
@@ -69,25 +69,25 @@ minimize.GP <- function(gp, rng, x0, splinefuns = NULL) {
 ##' @param psi spatial corr
 ##' @param x Name of variable to plot on X axis
 ##' @param rng range
-##' @param splinefcns spline functions
+##' @param splinefuns spline functions
 ##' 
 ##' @author Michael Dietze 
-gpeval <- function(xnew, k, mu, tau, psi, x, rng, splinefcns) {
+gpeval <- function(xnew, k, mu, tau, psi, x, rng, splinefuns) {
   
   ## second calc value
   S12 <- sapply(seq_along(k), function(i) {
     tau * exp(-sum(psi * (xnew - x[i, ]) ^ 2))
   })
   yprime <- mu + sum(S12 * k)
   
-  if (!is.null(splinefcns)) {
+  if (!is.null(splinefuns)) {
     ## add trend surface back on
     y0 <- splinefuns[[length(xnew) + 1]]
     f <- sapply(seq_along(xnew), function(j) {
       splinefuns[[j]](xnew[j])
```

Comment:
bug 2: these were null (unless something by the same name was found in the global environment) because in gpeval the parameter was `splinefcns`not `splinefuns`.